serverlessworkflow/modules/ROOT/pages/getting-started/cncf-serverless-workflow-specification-support.adoc

```diff
@@ -1 +1,368 @@
+
+For additional functions, the specification supports the `custom` type. Therefore, Kogito added support for `sysout` and `java` tyes. See <<func_sysout>> and <<func_java>> sections for more information.
+
+.Workflow Functions implementation status detail
+[cols="1,1,1"]
+|===
+| Function | Status | Spec Reference
+
+| rest
+| emoji:full_moon[]
+| {spec_doc_url}#using-functions-for-restful-service-invocations[Using Functions for RESTful Service Invocations]
+
+| rpc
+| emoji:full_moon[]
+| {spec_doc_url}#using-functions-for-rpc-service-invocations[Using Functions for RPC Service Invocations]
+
+| expression
+| emoji:full_moon[]
+| {spec_doc_url}#using-functions-for-expression-evaluation[Using Functions for Expression Evaluation]
+
+| asyncapi
+| emoji:construction[]
+| {spec_doc_url}#using-functions-for-async-api-service-invocations[Using Functions for Async API Service Invocations]
+
+| graphql
+| emoji:construction[]
+| {spec_doc_url}#using-functions-for-graphql-service-invocations[Using Functions for GraphQL Service Invocations]
+
+| odata
+| emoji:construction[]
+| {spec_doc_url}#using-functions-for-odata-service-invocations[Using Functions for OData Service Invocations]
+
+| custom
+| emoji:full_moon[]
+| {spec_doc_url}#defining-custom-function-types[Defining custom function types]. Kogito supports `java` and `sysout` custom functions.
+|===
+
+[#func_sysout]
+==== Sysout Custom Function
+
+You can use this `custom` function type for debugging reasons or quick demonstrations:
+
+.sysout function definition
+[source,json]
+----
+{
+  "functions": [
+    {
+      "name": "printMessage",
+      "type": "custom",
+      "operation": "sysout"
+    }
+  ]
+}
+----
+
+Later in the `state` definition, you can call it this way:
+
+.Example of a function reference within a state
+[source,json]
+----
+{
+  "states": [
+    {
+      "name": "myState",
+      "type": "operation",
+      "actions": [
+        {
+          "name": "printAction",
+          "functionRef": {
+            "refName": "printMessage",
+            "arguments": {
+              "message": "."
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
+----
+
+You should see the data output in the console.
+
+[#func_java]
+==== Java Custom Function
+
+Kogito supports calling Java functions within the Maven project, in which you define the workflow. For example, you can declare your functions like this:
+
+.Example of a Java function declaration
+[source,json]
+----
+{
+  "functions": [
+    {
+      "name": "myFunction", <1>
+      "type": "custom", <2>
+      "operation": "service:java:com.acme.MyInterfaceOrClass::myMethod" <3>
+    }
+  ]
+}
+----
+
+1. The function name
+2. The `custom` function type
+3. The Kogito custom operation definition. `service` is the reserved operation keyword followed by the other keyword `java`. The next portion is the FQDN of the interface or implementation class, followed by the method name.
```

Comment:
Maybe it worth to describe what for FQDN stands for on the first time it appears.